src/Fossa/API/Types.hs

```diff
@@ -236,8 +237,20 @@ instance ToJSON IssueRule where
 instance Pretty Issues where
   pretty = renderedIssues
 
+newtype OrgId = OrgId Int
+  deriving (Eq, Ord)
+
+instance Show OrgId where
+  show (OrgId orgId) = show orgId
+
+instance FromJSON OrgId where
+  parseJSON = fmap OrgId . parseJSON
+
+instance ToJSON OrgId where
+  toJSON (OrgId orgId) = toJSON orgId
```

Comment:
You can probably use `deriving (FromJSON, ToJSON)`  I would test it out, though, to make sure it derives how you expect it.  I _think_ HLS will tell you which extension you need to make this work, but we have other examples in the codebase.  Maybe grep for `deriving.*FromJSON`

Reply:
I think it's `GeneralizedNewtypeDeriving`, docs here: https://github.com/fossas/fossa-cli/blob/6c6fda19616b8141a1bd2a0fe39aeb48abcff068/src/Discovery/Archive.hs#L59-L67

Reply:
Yep, perfect.

```
Prelude Fossa.API.Types Data.Aeson> encode (OrgId 42)
"42"
Prelude Fossa.API.Types Data.Aeson> decode "42" :: Maybe OrgId
Just 42
```

Reply:
Nice.